<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogsWindow</class>
 <widget class="QMainWindow" name="LogsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1024</width>
      <height>686</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>984</width>
        <height>630</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendar">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>740</width>
        <height>340</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
alternate-background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="firstDayOfWeek">
       <enum>Qt::Monday</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>420</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_currentdate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string>Selected Date:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openLogs">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>525</y>
        <width>200</width>
        <height>85</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string> LOGS</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/logs.png</normaloff>:/new/icons/logs.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_dateselector">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string>Date Selector:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>495</y>
        <width>1120</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openfaults">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>525</y>
        <width>200</width>
        <height>85</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string> FAULTS</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/new/icons/warning.png</normaloff>:/new/icons/warning.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_info_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>984</width>
        <height>630</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openfile">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>533</y>
        <width>175</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>OPEN FILE</string>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>28</y>
        <width>944</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_info_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>984</width>
        <height>630</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>28</y>
        <width>944</width>
        <height>480</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openfile_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>533</y>
        <width>175</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>OPEN FILE</string>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_home_button">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/icons/home-button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_heading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string> LOGS/FAULTS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_button">
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>0</y>
      <width>190</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Date">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>730</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Time">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>720</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>934</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/icons/back-arrow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>label_heading</zorder>
   <zorder>label_Date</zorder>
   <zorder>label_Time</zorder>
   <zorder>label_button</zorder>
   <zorder>pushButton_home_button</zorder>
   <zorder>pushButton_return</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
